README

```diff
@@ -34,3 +34,7 @@ stable linux kernel. The repository currently has a few branches.
 
 * opinicus-4.9.13_0			Our current kernel based on 4.9.13 tag
 * opinicus-4.10.1_0			An experimental future kernel
+
+dependencies
+------------
+* debian: device-tree-compiler gcc-arm-linux-gnueabihf u-boot-tools
```

Comment:
I disagree, as these dependencies are now already wrong. And listing them in multiple locations just gives more locations where they can be outdated/conflicting or wrong.

We want to depend specifically on gcc 4.9 right now. Ubuntu is shipping a newer gcc for ARM per default, and that is giving some issues right now.